.github/workflows/test.yml

```diff
@@ -119,51 +135,69 @@ jobs:
       MYPY_FORCE_TERMINAL_WIDTH: 200
       # Pytest
       PYTEST_ADDOPTS: --color=yes
+
     steps:
     - uses: actions/checkout@v4
-    - uses: actions/setup-python@v5
-      with:
-        python-version: ${{ matrix.python }}
-        architecture: ${{ matrix.arch }}
+
     - name: Debug build
       if: ${{ matrix.debug_build }}
       run: |
         PYTHONVERSION=${{ matrix.python }}
         PYTHONDIR=~/python-debug/python-$PYTHONVERSION
         VENV=$PYTHONDIR/env
         ./misc/build-debug-python.sh $PYTHONVERSION $PYTHONDIR $VENV
+        # TODO: does this do anything? env vars aren't passed to the next step right
```

Comment:
Filed https://github.com/python/mypy/issues/17819 , I'll also fix the suit typo while I'm there